content/pt/docs/concepts/instrumentation/libraries.md

```diff
@@ -0,0 +1,465 @@
+---
+title: Bibliotecas
+description: Aprenda como adicionar instrumentação nativa à sua biblioteca.
+weight: 40
+default_lang_commit: a570a00c7a238ffe26528d7bfb20efdbaf939c39
+---
+
+O OpenTelemetry fornece [bibliotecas de instrumentação][] para várias
+bibliotecas, geralmente feitas por meio de _hooks_ de biblioteca ou
+_monkey-patching_ do código da biblioteca.
+
+A instrumentação nativa de bibliotecas com OpenTelemetry oferece melhor
+observabilidade e experiência para desenvolvedores, eliminando a necessidade de
+as bibliotecas exporem e documentarem hooks:
+
+- hooks de logging personalizados podem ser substituídos por APIs OpenTelemetry
+  comuns e fáceis de usar, os usuários utilizarão somente o OpenTelemetry
+- rastros, logs e métricas do código da biblioteca e da aplicação são
+  correlacionados e coerentes
+- convenções comuns permitem que os usuários obtenham uma telemetria semelhante
+  e consistente dentro da mesma tecnologia e entre bibliotecas e linguagens
+- sinais de telemetria podem ser ajustados (filtrados, processados, agregados)
+  para diversos cenários de consumo usando uma grande variedade de pontos de
+  extensibilidade bem documentados do OpenTelemetry.
```

Comment:
Curious: why list items here start in lowercase whereas other lists have items starting with uppercase?

Reply:
@theletterf good question. 
In the documentation has lists with uppercase and downcase. 
https://opentelemetry.io/docs/concepts/instrumentation/libraries/

Reply:
Hmm, we might want to fix the English version too, then